portal-impl/src/portal.properties

```diff
@@ -5938,11 +5938,13 @@
     setup.database.driverClassName[sybase]=com.sybase.jdbc4.jdbc.SybDriver
 
     #
+    # Env: LIFERAY_SETUP_PERIOD_DATABASE_PERIOD_JAR_PERIOD_CHECKSUM_OPENBRACKET_COM_PERIOD_MYSQL_PERIOD_CJ_PERIOD_JDBC_PERIOD__UPPERCASED_RIVER_CLOSEBRACKET_
     # Env: LIFERAY_SETUP_PERIOD_DATABASE_PERIOD_JAR_PERIOD_NAME_OPENBRACKET_COM_PERIOD_MYSQL_PERIOD_CJ_PERIOD_JDBC_PERIOD__UPPERCASED_RIVER_CLOSEBRACKET_
     # Env: LIFERAY_SETUP_PERIOD_DATABASE_PERIOD_JAR_PERIOD_URL_OPENBRACKET_COM_PERIOD_MYSQL_PERIOD_CJ_PERIOD_JDBC_PERIOD__UPPERCASED_RIVER_CLOSEBRACKET_
     #
     setup.database.jar.name[com.mysql.cj.jdbc.Driver]=mysql.jar
     setup.database.jar.url[com.mysql.cj.jdbc.Driver]=https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.21/mysql-connector-java-8.0.21.jar
+    setup.database.jar.checksum[com.mysql.cj.jdbc.Driver]=53fd3a0887667aae7e40a3439fff2d03d93ec4c2
```

Comment:
@dantewang The property key here is "setup.database.jar.checksum", but corresponding changes in PropsKeys is public static final String SETUP_DATABASE_JAR_SHA1 ="setup.database.jar.sha1";

They are not consistent.